packages/react-components/src/components/VideoTile.tsx

```diff
@@ -156,32 +156,34 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           </Stack>
         )}
 
-        <Stack
-          horizontal
-          className={mergeStyles(
-            isVideoRendered ? videoHint : disabledVideoHint,
-            // when video is being rendered, the displayName has a grey-ish background, so no use of theme
-            { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary },
-            styles?.displayNameContainer
-          )}
-        >
-          <Stack>
-            {displayName && (
-              <Text
-                className={mergeStyles(displayNameStyle, {
-                  color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary
-                })}
-              >
-                {displayName}
-              </Text>
+        {(displayName || showMuteIndicator) && (
+          <Stack
+            horizontal
+            className={mergeStyles(
+              isVideoRendered ? videoHint : disabledVideoHint,
+              // when video is being rendered, the displayName has a grey-ish background, so no use of theme
+              { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary },
```

Comment:
If this part is not removable, assign it with a const

And also wrap both mergeStyles with useMemo()

Since videotiles will be rendering a lot in the app, any perf improvement counts

Reply:
const nametagColorOverride = { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary };

Then

const displayNameContainerStyle = useMemo(() => mergeStyles(...));